lib/standards/html-elms.js

```diff
@@ -914,7 +923,8 @@ const htmlElms = {
         contentTypes: ['embedded', 'phrasing', 'flow']
       }
     },
-    allowedRoles: ['application']
+    allowedRoles: ['application'],
+    chromiumRole: 'video'
```

Comment:
Same here. Why is it `Audio` but not `Video`?

Reply:
Same answer - it's inconsistently reported in the browser.